src/robusta/core/model/nodes.py

```diff
@@ -27,56 +27,7 @@ def __eq__(self, other):
         if not isinstance(other, NodeInfo):
             return NotImplemented
 
-        ignored_fields = ["deleted", "node_creation_time"]  # node_creation_time never changes
+        ignored_fields = ["deleted"]
```

Comment:
any reason why we stopped ignoring this? i think it should be in nodeInfo but i was wondering if we saw the time changes.

Reply:
Yeah, the time changes when you upgrade the node.
I wrote it in the comment